wasp-ai/main.wasp

```diff
@@ -41,13 +42,17 @@ app waspAi {
     userEntity: User,
     externalAuthEntity: SocialLogin,
     methods: {
+      gitHub: {
```

Comment:
@vincanger when you deploy an update make sure to put the Github env variables

Reply:
we're not using github login at all right now, plus I was going to leave deployment to you guys